components/console-backend-service/integration/main_test.go

```diff
@@ -0,0 +1,158 @@
+package integration
+
+import (
+	"fmt"
+	"github.com/kyma-project/kyma/components/console-backend-service/integration/graphql"
+	"github.com/kyma-project/kyma/components/console-backend-service/pkg/app"
+	"github.com/stretchr/testify/require"
+	"github.com/vrischmann/envconfig"
+	rbacv1 "k8s.io/api/rbac/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apiserver/pkg/authentication/authenticator"
+	"k8s.io/apiserver/pkg/authentication/user"
+	rbacClient "k8s.io/client-go/kubernetes/typed/rbac/v1"
+	"k8s.io/client-go/rest"
+	"net"
+	"net/http"
+	"os"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"sync"
+	"testing"
+	"time"
+)
+
+var (
+	restConfig    *rest.Config
+	gqlEndpoint   string
+	rbacClientset *rbacClient.RbacV1Client
+)
+
+func TestMain(m *testing.M) {
+	// setup and start kube-apiserver
+	var err error
+	environment := &envtest.Environment{
+		KubeAPIServerFlags: append([]string{"--authorization-mode=RBAC"}, envtest.DefaultKubeAPIServerFlags...),
+	}
+	restConfig, err = environment.Start()
+	if err != nil {
+		panic(err)
+	}
+
+	_, err = envtest.InstallCRDs(restConfig, envtest.CRDInstallOptions{
+		Paths:              []string{"../../../resources/cluster-essentials/templates/crds"},
```

Comment:
Also all BackendModule CRs should be applied as well to enable all CBS modules

Reply:
👍 I would like to add them gradually while migrating tests.